tpm/tpm.go

```diff
@@ -1005,3 +1005,16 @@ func PcrReset(rw io.ReadWriter, pcrs []int) error {
 	}
 	return nil
 }
+
+// ForceClear is normally used by firmware but on some platforms
+// vendors got it wrong and didn't call TPM_DisableForceClear.
+func ForceClear(rw io.ReadWriter) error {
+	in := []interface{}{}
+	out := []interface{}{}
+	_, err := submitTPMRequest(rw, tagRQUCommand, ordForceClear, in, out)
+	if err != nil {
```

Comment:
this line and the following could simply be `return err`

Reply:
Agreed. Please fix.

Reply:
Done